prover/src/lib.rs

```diff
@@ -522,6 +523,7 @@ pub trait Prover {
         composition_poly_trace: CompositionPolyTrace<E>,
         num_constraint_composition_columns: usize,
         domain: &StarkDomain<Self::BaseField>,
+        partition_size: usize,
```

Comment:
Why do we need to pass `partition_size` into this function? Shouldn't we be able to compute it from the info in `self`?

Reply:
just a leftover, changed it now